crates/neon/src/context/mod.rs

```diff
@@ -585,19 +586,18 @@ impl<'a> ContextInternal<'a> for ExecuteContext<'a> {
 impl<'a> Context<'a> for ExecuteContext<'a> {}
 
 /// An execution context of a scope created by [`Context::compute_scoped()`](Context::compute_scoped).
-pub struct ComputeContext<'a, 'outer> {
+pub struct ComputeContext<'a> {
     env: Env,
     phantom_inner: PhantomData<&'a ()>,
-    phantom_outer: PhantomData<&'outer ()>,
```

Comment:
This was required before to ensure that `ComputeContext` couldn't accidentally outlive the outer context. However, `compute_scoped` now includes a `'a: 'b` bound that provides the same guarantee.